Knossos.NET/Models/Mod.cs

```diff
@@ -348,127 +348,221 @@ public List<string> GetModFlagList(bool overrideSettings = false)
+                        // ==  /  ~  /  <=  /  >=
+
+                        //==
+                        //Note: There is no need to compare equals to other equals
+                        equal.ForEach(eq => {
+                            //Remove any "rev" that can be sastified by this equal
+                            rev.ToList().ForEach(r =>
+                            {
+                                if (SemanticVersion.SastifiesDependency(r.version, eq.version))
                                 {
-                                    temp.Remove(d);
+                                    rev.Remove(r);
                                 }
-                                else
+                            });
+                            //Remove any "lower" that can be sastified by this equal
+                            lower.ToList().ForEach(l =>
+                            {
+                                if (SemanticVersion.SastifiesDependency(l.version, eq.version))
                                 {
-                                    if (d.version.Contains(">="))
-                                    {
-                                        equalOrHigher.Add(d);
-                                    }
-                                    else
-                                    {
-                                        if (d.version.Contains("~"))
-                                        {
-                                            revisions.Add(d);
-                                        }
-                                        else
-                                        {
-                                            equal.Add(d);
-                                        }
-                                    }
+                                    lower.Remove(l);
                                 }
-                            }
-                            //Equal determine if equalOrhigher or revisions are included here, if so remove them
-                            foreach (var eq in equal)
+                            });
+                            //Remove any "higher" that can be sastified by this equal
+                            higher.ToList().ForEach(h =>
                             {
-                                equalOrHigher.ForEach(eqOrHigh =>
+                                if (SemanticVersion.SastifiesDependency(h.version, eq.version))
                                 {
-                                    if (SemanticVersion.SastifiesDependency(eqOrHigh.version, eq.version))
-                                    {
-                                        temp.Remove(eqOrHigh);
-                                    }
-                                });
-                                revisions.ForEach(revs =>
+                                    higher.Remove(h);
+                                }
+                            });
+                        });
+
```

Comment:
I didn't see a section where `==` dependencies filter out `~` dependencies, as you said in the main comment above.

Reply:
Equals are checked on https://github.com/KnossosNET/Knossos.NET/blob/8ff6e6b167b177d1a61e84f1e2bc0f9dfbfffc8d/Knossos.NET/Models/Mod.cs#L433